packages/firebase_remote_config/firebase_remote_config/ios/Classes/FLTFirebaseRemoteConfigPlugin.m

```diff
@@ -161,7 +192,21 @@ - (void)fetchAndActivate:(id)arguments withMethodCallResult:(FLTFirebaseMethodCa
   [remoteConfig fetchAndActivateWithCompletionHandler:^(
                     FIRRemoteConfigFetchAndActivateStatus status, NSError *error) {
     if (error != nil) {
-      result.error(nil, nil, nil, error);
+      if (error.code == 999 && _fetchAndActivateRetry == false) {
+        // Note: see issue for details: https://github.com/firebase/flutterfire/issues/6196
+        // Only calling once as the issue noted describes how it works on second retry
+        // Issue appears to indicate the error code is: 999
```

Comment:
Should the second call be delayed by a couple of milliseconds?

Reply:
Maybe something the user can configure?

Reply:
I'm not in favour of either at the moment. I think if a further delay is needed, we can implement as and when it is needed. It is an underlying implementation helper that is only activated on this specific error code to make sure the values are fetched and activated so I'd prefer to hide this from the user.

Reply:
Okay I guess we'll see if the error is fixed with that :)